data/meeting_minutes.yml

```diff
@@ -4,6 +4,14 @@ years:
   - 2018
 items:
    marketing_committee:
+        -
```

Comment:
There is one too many spaces before the dash on this line, which is likely causing the breakage. The way this file is written is super picky with the consistency of spacing